src/App.js

```diff
@@ -1,8 +1,14 @@
-import React from 'react';
-import 'bootstrap/dist/css/bootstrap.min.css';
-
-function App() {
-  return <h1>Hello Wors</h1>;
+import React, { Component } from 'react';
+import Header from './Components/Header/Header';
+class App extends Component {
+  render() {
+    return (
+      <>
+        <Header />
+        <div className="container">content</div>
+      </>
+    );
+  }
```

Comment:
This have to be a functional component, no need to use Component here.
Please refactor